src/test/Common.hs

```diff
@@ -73,7 +73,7 @@ runContract :: FilePath -> Maybe ContractName -> EConfig -> IO Campaign
 runContract f c cfg =
   flip runReaderT cfg $ do
     g <- getRandom
-    (v, w, ts, d, txs) <- prepareContract cfg (f :| []) c g
+    (v, _, _, w, ts, d, txs) <- prepareContract cfg (f :| []) c g
     -- start ui and run tests
     campaign (pure ()) v w ts d txs
```

Comment:
now that we have a bunch of things from `prepareContract` can we make a custom type for `prepareContract` and `campaign` and `ui` to ingest the custom type instead of adding a new parameter every time? feel free to make an issue and refactor in a separate PR.

Reply:
Create a new issue to track down this.